src/main/java/org/openstreetmap/atlas/geography/atlas/builder/RelationBean.java

```diff
@@ -31,6 +33,26 @@ public void addItem(final Long identifier, final String role, final ItemType ite
         this.memberTypes.add(itemType);
     }
 
+    @Override
+    public boolean equals(final Object other)
+    {
+        if (Objects.nonNull(other) && other instanceof RelationBean)
```

Comment:
Nit: nonNull is not needed - `instanceof` will perform the null check under the covers.